b2sdk/transfer/emerge/executor.py

```diff
@@ -409,6 +427,11 @@ def _match_unfinished_file_if_possible(
             if encryption is not None and encryption != file_.encryption:
                 logger.debug('Rejecting %s: encryption mismatch', file_.file_id)
                 continue
+            
+            # FIXME as above - shoule this be handled in the same way as encryption?
```

Comment:
yes, it's clearly a different upload intent than the original

Reply:
@ppolewicz can you please elaborate? I am not sure what you meant by this comment

Reply:
if the cache control setting is different than in the given unfinished file upload, then we should reject it as a candidate for continuation (emit a debug log and `continue`